pkg/stages/stages.go

```diff
@@ -340,6 +339,28 @@ func GetInitrdStage(_ values.System, logger types.KairosLogger) ([]schema.Stage,
 			return []schema.Stage{}, err
 		}
 
+		if config.DefaultConfig.Fips {
+			// Add dracut fips support
+			if sys.Distro == values.Ubuntu {
+				return nil, fmt.Errorf("FIPS is not supported on Ubuntu without a PRO account and extra packages. See https://canonical-ubuntu-pro-client.readthedocs-hosted.com/en/latest/tutorials/create_a_fips_docker_image.html for more info")
```

Comment:
wasn't the edge team using it with Ubuntu Pro?

Reply:
Yes, if they want fips with Ubuntu pro, they will need to build it themselves by following the example we got there

Unfortunately there is no easy way of setting this for Ubuntu as it needs that pro account so this is the first step to support generic fips building for other flavors, hence the link. I guess that if Ubuntu and fips is selected we could try to enable Ubuntu pro repos and install the feature and fail in there early?

I'm also wondering if we should set the fips value in the kairos-release so we can use the automatic bootargs to set the fips=1 cmdline automatically on boot so this is more automatic